sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/CreateNewContactPage.java

```diff
@@ -21,17 +21,41 @@
 import org.openqa.selenium.By;
 
 public class CreateNewContactPage {
+
   public static final By FIRST_NAME_OF_CONTACT_PERSON_INPUT =
       By.cssSelector(".v-window [location='firstName'] input");
   public static final By LAST_NAME_OF_CONTACT_PERSON_INPUT =
       By.cssSelector(".v-window [location='lastName'] input");
+  public static final By DATE_OF_BIRTH_YEAR_COMBOBOX =
+      By.cssSelector(".popupContent #birthdateYYYY input+div");
+  public static final By DATE_OF_BIRTH_MONTH_COMBOBOX =
+      By.cssSelector(".popupContent #birthdateMM input+div");
+  public static final By DATE_OF_BIRTH_DAY_COMBOBOX =
+      By.cssSelector(".popupContent #birthdateDD input+div");
   public static final By SEX_COMBOBOX =
       By.cssSelector(".v-window [location='sex'] [role='combobox'] div");
+  public static final By NATIONAL_HEALTH_ID_INPUT = By.cssSelector("#nationalHealthId");
+  public static final By PASSPORT_NUMBER_INPUT = By.cssSelector("#passportNumber");
```

Comment:
Please add `.v-window` or `.popupContent` if this is on popup.

Reply:
there's no need, but I'll keep in mind for the next locators located in popups 👍